API-strategie-modulen/API-strategie-mod-geo/request-response.md

```diff
@@ -58,17 +48,25 @@ A simple spatial filter can be supplied as a bounding box. This is a common way
   </p>
   <h4 class="rulelab">How to test</h4>
   <ul>
-    <li>Issue an HTTP GET request to the API, including the <code>bbox</code> query parameter.</li>
-    <li>Validate that a document was returned with a status code 200.</li>
+    <li>Issue an HTTP GET request to the API, including the <code>bbox</code> query parameter and <code>bbox-crs</code> header containing a CRS supported by the provider. If the provider supports CRS84 then the <code>bbox-crs</code> header may be omitted.</li>
+    <li>Validate that a response with status code 200 is returned.</li>
     <li>Verify that only features that have a spatial geometry that intersects the bounding box are returned as part of the result set.</li>
   </ul>
 </div>
 
+TO DO 
+Bovenstaande voorbeeld is redundant met het voorbeeld bij API-GEO-10. Het lijkt me goed om hier het voorbeeld van bbox te gebruiken en bij API-GEO-10 een ander voorbeeld, of vanuit API-GEO-10 te verwijzen naar dit voorbeeld of andersom.
```

Comment:
ik denk dat je in API-GEO-2 een voorbeeld zou kunnen gebruiken van bbox parameter, en in API-GEO-10 eentje met bbox EN bbox-crs parameters. En of dat dan dezelfde coordinaten zijn maakt niet zo uit.